### PR TITLE
Popovers

### PR DESCRIPTION
**Pull request by evrowe**
## Description
Adds `core-popover` component, a demo page for the dummy app and some tests.

Addresses #21.

These are the changes included:

- ✨ `core-popover`
- ✨ `core-popover/content`
- 📝 Examples of new components in dummy app
- ✏️ Fixed a docs error
- ✅ Tests for new components

## Tests
- [x] I added tests for changes I made
- [x] All tests are _definitely_ passing

### CODE REVIEW

**Comment by evrowe** [comment]
Dan I figure we can get this merged in now and then we can clean up the CSS either before or after the 1.0 release

File: app/styles/ember-radical/component-structures/_popovers.scss [hunk 1]
```diff
@@ -0,0 +1,202 @@
+// ========================================================
+// Popover Structural Styles
+// @TODO: Rework the layout styles to use flexbox
+// ========================================================
+
+$small-popover-width:     243px;
```
Comment (bigmoves) [inline app/styles/ember-radical/component-structures/_popovers.scss]: Should we move these to the variables file under Popovers?

Comment (evrowe) [inline app/styles/ember-radical/component-structures/_popovers.scss]: Well, I think the original idea was that Dan had a plan for reworking all of this CSS with flexbox anyway, so I don't really care that much about the contents of this file at this point, unless Dan isn't going to get to the rework until after 1.0.

@DHedgecock ?

Comment (bigmoves) [inline app/styles/ember-radical/component-structures/_popovers.scss]: Ah ok

Comment (bigmoves) [inline app/styles/ember-radical/component-structures/_popovers.scss]: just merge it™

**Pull request merged**